apps/api/src/auth/services/ability/ability.service.ts

```diff
@@ -24,21 +24,24 @@ export class AbilityService {
     REGULAR_USER: [
       { action: ["create", "read", "sign"], subject: "UserWallet", conditions: { userId: "${user.id}" } },
       { action: "read", subject: "User", conditions: { id: "${user.id}" } },
+      { action: "manage", subject: "DeploymentSetting", conditions: { userId: "${user.id}" } },
+
       { action: ["create", "read", "delete"], subject: "StripePayment" },
       { action: "create", subject: "VerificationEmail", conditions: { id: "${user.id}" } },
-      { action: "manage", subject: "DeploymentSetting", conditions: { userId: "${user.id}" } },
       { action: "manage", subject: "Alert", conditions: { userId: "${user.id}" } },
       { action: "manage", subject: "NotificationChannel", conditions: { userId: "${user.id}" } }
     ],
     REGULAR_PAYING_USER: [
       { action: ["create", "read", "sign"], subject: "UserWallet", conditions: { userId: "${user.id}" } },
       { action: "read", subject: "User", conditions: { id: "${user.id}" } },
+      { action: "manage", subject: "DeploymentSetting", conditions: { userId: "${user.id}" } },
+
       { action: ["create", "read", "delete"], subject: "StripePayment" },
       { action: "create", subject: "VerificationEmail", conditions: { id: "${user.id}" } },
-      { action: "manage", subject: "DeploymentSetting", conditions: { userId: "${user.id}" } },
-      { action: "manage", subject: "ApiKey", conditions: { userId: "${user.id}" } },
       { action: "manage", subject: "Alert", conditions: { userId: "${user.id}" } },
-      { action: "manage", subject: "NotificationChannel", conditions: { userId: "${user.id}" } }
+      { action: "manage", subject: "NotificationChannel", conditions: { userId: "${user.id}" } },
+
+      { action: "manage", subject: "ApiKey", conditions: { userId: "${user.id}" } }
```

Comment:
just re-arranged this for now to make it more obvious that roles build up permissions gradually